src/__tests__/PartialContainer.spec.ts

```diff
@@ -152,6 +152,43 @@ describe("PartialContainer", () => {
       });
     });
 
+    describe("provide service using provideValue", () => {
+      const dependenciesContainer = Container.provides(Injectable("TestService", () => "old service"));
+
+      describe("and the new Service does not override", () => {
+        const partialContainer = new PartialContainer({}).providesValue("NewTestService", "new service");
+        expect(dependenciesContainer.provides(partialContainer).get("NewTestService")).toEqual("new service");
+      });
+
+      describe("and the new Service does override", () => {
+        const partialContainer = new PartialContainer({}).providesValue("TestService", "new service");
+        expect(dependenciesContainer.provides(partialContainer).get("TestService")).toEqual("new service");
+      });
+    });
+
+    describe("provide service using provideClass", () => {
+      const dependenciesContainer = Container.provides(Injectable("TestService", () => "old service"));
+
+      class NewTestService {
+        static dependencies = ["TestService"] as const;
+        constructor(public testService: string) {}
+      }
+
+      describe("and the new Service does not override", () => {
```

Comment:
please replace `describe` with `test` here and in other case. the expectation should be inside `test` block.